Firmware/mmu2_reporting.cpp

```diff
@@ -359,4 +372,68 @@ void ScreenClear(){
     lcd_clear();
 }
 
+struct TuneItem {
+    uint8_t address;
+    uint8_t minValue;
+    uint8_t maxValue;
+} __attribute__((packed));
+
+static const TuneItem TuneItems[] PROGMEM = {
+  { (uint8_t)Register::Selector_sg_thrs_R, 1, 4},
+  { (uint8_t)Register::Idler_sg_thrs_R, 4, 7},
+};
+
+static_assert(sizeof(TuneItems)/sizeof(TuneItem) == 2);
+
+typedef struct
```

Comment:
let's make it a C++ struct

Reply:
Do I understand correctly you mean to drop `typedef`?

Reply:
Done